src/content/collaboration/documents/snapshot-compare.mdx

```diff
@@ -0,0 +1,203 @@
+---
+title: Compare Snapshots of your Document
+meta:
+  title: Snapshot Compare extension | Tiptap Editor Docs
+  description: Compare snapshots of versions of your documents to see changes made between two versions.
+  category: Collaboration
+extension:
+  name: SnapshotCompare
+  description: 'Compare snapshots of versions of your documents to see changes made between two versions.'
+  type: extension
+  icon: FileStack
+  isPro: true
+  isNew: true
+  isCloud: true
+---
+
+import { Callout } from '@/components/ui/Callout'
```

Comment:
This is obviously the bulk of the documentation written so far, but it definitely needs reorganization